daemon/anonymous-statistics.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env sh
+
+# Valid actions:
+# - KICKSTART   - installed using the kickstart.sh script
+# - KICKSTART64 - installed using the kickstart-static64.sh script
+# - MAKESELF    - installed by running a makeself package
+# - INSTALL     - installed using netdata-installer.sh
+# - UPDATE      - updated using netdata-updater.sh
+# - FATAL       - netdata exited due to a fatal condition
+# - START       - netdata started
+
+ACTION="${1}"
+ACTION_RESULT="${2}"
+ACTION_DATA="${3}"
+
+# -------------------------------------------------------------------------------------------------
+# check opt-out
+
+OPTOUT=
+if [ -z "${NETDATA_CONFIG_DIR}" ]; then
+    for dir in "$(netdata -W get global 'config directory' '/etc/netdata' 2>/dev/null)" "/etc/netdata" "/opt/netdata/etc/netdata"; do
+        if [ -f "${dir}/.opt-out-from-anonymous-statistics" ]; then
+            OPTOUT="$(cat "${dir}/.opt-out-from-anonymous-statistics")"
+            break
+        fi
+    done
+else
+    OPTOUT="$(cat "${NETDATA_CONFIG_DIR}/.opt-out-from-anonymous-statistics")"
+fi
+
+if [ "${OPTOUT}" = "all" ] || ( [ "${OPTOUT}" = "usage" ] && [ "${ACTION}" = "START" ] ); then
```

Comment:
There is no need to spawn a subshell with `(...)` when the same behavior can be accomplished with `{ ..; }`.